jenkins/PerfCI_Chaos/03_PerfCI_Choas_Windows_VMs_Deployment/Jenkinsfile

```diff
@@ -197,8 +224,8 @@ END
                                         -e IBM_BUCKET='${IBM_BUCKET}' \
                                         -e IBM_KEY='${IBM_KEY}' \
                                         -e GOOGLE_DRIVE_PATH='${GOOGLE_DRIVE_PATH}' \
-                                        -e GOOGLE_DRIVE_CREDENTIALS_FILE='${GOOGLE_DRIVE_CREDENTIALS_FILE}' \
-                                        -e GOOGLE_DRIVE_TOKEN_FILE='${GOOGLE_DRIVE_TOKEN_FILE}' \
+                                        -e GOOGLE_DRIVE_CREDENTIALS_FILE='/tmp/credentials.json' \
+                                        -e GOOGLE_DRIVE_TOKEN_FILE='/tmp/token.json' \
```

Comment:
Where do these filenames  come from?  In any case, use mktemp-generated temporaries rather than fixed filenames which could be subject to various attacks.

Reply:
As I said before it is internal Jenkins under VPN

Reply:
Those file come from secrets and cannot be shared with anyone:
https://github.com/redhat-performance/benchmark-runner/blob/5e1a0e52905bf51e28227f168eb1d3499707e042/jenkins/PerfCI_Chaos/03_PerfCI_Choas_Windows_VMs_Deployment/Jenkinsfile#L23-L24

Reply:
But it looks like you're copying them into local files in /tmp?

Reply:
Yes, this is the only way that I found that Jenkins works, otherwise it failed to read it.

Reply:
I will put it in secret

Reply:
Pls take a look now, its fully secured